validation/static/styles.css

```diff
@@ -5,6 +5,10 @@ main {
   margin: 1em auto;
 }
 
+.main_container {
+  width: 90%;
```

Comment:
Oh dear! I wonder why this was necessary!

Reply:
I wanted the cards to be a little wider so things wouldn't jump around, which meant making the main container larger so the card could be 100% the width of the container. I don't want it to span the full width of the screen, hence 90%.